termino/termino.py

```diff
@@ -52,25 +52,25 @@ def __init__(self, bot):
         self.announce_startup = self.bot.loop.create_task(self._announce_start())
         self._first_connect = False
 
-    def cog_unload(self):
+    async def cog_unload(self):
         global shutdown, restart
         if shutdown:
             try:
-                self.bot.remove_command("shutdown")
+                await self.bot.remove_command("shutdown")
             except Exception as e:
                 log.info(e)
-            self.bot.add_command(shutdown)
+            await self.bot.add_command(shutdown)
         if restart:
             try:
-                self.bot.remove_command("restart")
+                await self.bot.remove_command("restart")
             except Exception as e:
                 log.info(e)
-            self.bot.add_command(restart)
+            await self.bot.add_command(restart)
         if 719988449867989142 in self.bot.owner_ids:
             with contextlib.suppress(KeyError):
-                self.bot.remove_dev_env_value(self.__class__.__name__.lower())
-        self.startup_task.cancel()
-        self.announce_startup.cancel()
+                await self.bot.remove_dev_env_value(self.__class__.__name__.lower())
+        await self.startup_task.cancel()
+        await self.announce_startup.cancel()
```

Comment:
`task.cancel` and `bot.remove_dev_env_value` are also a synchronous function so this will fail